tests/rptest/services/redpanda.py

```diff
@@ -126,6 +126,18 @@ def __repr__(self):
         return self.__str__()
 
 
+class NodeCrash(Exception):
+    def __init__(self, crashes):
+        self.crashes = crashes
+
+        # Not legal to construct empty
+        assert len(crashes)
+
+    def __str__(self):
+        example = f"<NodeCrash {self.crashes[0][0].name}: {self.crashes[0][1]}>"
+        return f"<NodeCrash ({len(self.crashes)}) {example}>"
```

Comment:
Also, based on the notation, it seems the returned string could indicate crashes on multiple nodes because of `<Node Crash ({len(self.crashes)}) ...>` but we only report the failed assert for a single node because of `<... {self.crashes[0][1]}>`.

Why not report failed asserts from all crashed nodes?

Reply:
The duplication of `<NodeCrash` is a typo on my part (fixed).

The reason I don't report all the nodes crash lines is to keep the exception string length manageable: __str__ on an exception is meant to output something that very roughly fits on a line.